test/wrb.test.js

```diff
@@ -304,17 +303,15 @@ contract("WitnetRequestBoard", ([
         "contract balance should have decreased after reporting dr request result by 1 eth",
       ).to.equal(true)
       expect(
-        ownerFinalBalance.eq(ownerInitialBalance
-          .add(ether("1")).sub(new BN(reportResultTx.receipt.gasUsed))
-        ),
+        ownerFinalBalance.gt(ownerInitialBalance),
```

Comment:
Why don't we need to consider the `gasUsed` anymmore?

Reply:
Not relevant to the actual test being performed: "contract balance should have decreased after reporting dr request result by 1 eth"